src/moin/templates/ajaxsearch.html

```diff
@@ -56,14 +56,25 @@ <h4>{{ _("Please check these tickets if they cover your issue:") }}</h4>
                     <span class="moin-suggestion-terms">{{ content_suggestions }}</span>
                 </p>
             {% endif %}
-
-            {% set count = [0] %}
             {% if results %}
                 <div class="moin-search-results">
-                    <table>
+                    {% set count = [0] %}
+                    <table
+                        {% set vars = {'old_acl_in': [None, None, None], 'old_permission': False} %}
                         {% for result in results %}
                             {% set fullname = result.namespace + '/' + result.name[0] if result.namespace else result.name[0] %}
-                            {% if user.may.read(fullname) %}
+                            {% set new_acl = result['acl'] if 'acl' in result else '' %}
```

Comment:
could this be `result.get('acl', '')` ?

Reply:
yes, fixed in my repo as the above caused traceback